examples/mpi/run.sh

```diff
@@ -2,7 +2,7 @@
 #SBATCH --account=ncemhub
 #SBATCH --qos=debug
 #SBATCH -C haswell
-#SBATCH --time=00:02:00
+#SBATCH --time=00:01:00
 #SBATCH -N 8
 
 home=/global/homes/p/psavery/
```

Comment:
When we get a real example we should update this to use the pipeline engine, so we will need to make sure that any mpi specfic logic in `run.py` incorporated into the command line.